pkg/util/kubeconfig_test.go

```diff
@@ -394,6 +394,107 @@ func TestDeleteKubeConfigContext(t *testing.T) {
 	}
 }
 
```

Comment:
how about moving all the kubeconfig related things to a package named kubeconfig ?